src/dune

```diff
@@ -8,6 +8,7 @@
   -w -projection-no-head-constant)
  (theories
   elpi
+  elpi_elpi
```

Comment:
The source of the issue seems to be the fact that Coq-Elpi uses two prefixes of logical paths `elpi` and `elpi_elpi`. So this line fixes the issue. However, I see this FIXME comment on the Coq-Elpi side: https://github.com/LPCIC/coq-elpi/blob/c13ecc9182c4344c00894d0523ccba1d5c03e6d8/elpi/dune#L2

@gares Can you confirm that this is the right fix? Or is it reasonable to fix the issue on the Coq-Elpi side?

Also CC Dune experts: @ejgallego @Alizter